### DISCUSSION TITLE
Add option to copy over non-javascript files when obfuscating a folder

### DISCUSSION THREAD

**Discussion by jmechevarria** [javascript-obfuscator/javascript-obfuscator#910] [Q&A]
This is more like a feature request, if I should post this somewhere else, please do let me know.

Is there or could there be an option (CLI arg) to include the rest of the files in a folder when obfuscating said folder's JS files?

If a have files **src/js/a.js**, **src/js/b.js** and **src/css/a.css**, I'd like to be able to do something like

`javascript-obfuscator src --output src-o --copy-rest`

so I end up with an obfuscated version of my JS files, alongside my non-JS files.

This would be very handy when dealing with lots of files.

Thank you

**Comment by JamieGrimwood** [comment]
Has this ever been looked at being implemented? Need something like this

**Comment by jeff-1984** [comment]
There is another npm package we can use for that : 
npm install copyfiles

In package.json, add this :

`
"scripts": {
    "obfuscate": "javascript-obfuscator ./src ./dist --output ./dist",
    "copy-rest": "copyfiles -u 1 src/**/*.html src/* */*.css src/app/statics/** dist"
  }
`


Then you can do :
npm run-script obfuscate
npm run-script copy-rest
